test/functional/feature_block.py

```diff
@@ -1201,33 +1186,30 @@ def get_tests(self):
             tx.vin.append(CTxIn(COutPoint(b.vtx[1].sha256, 0)))
             b = self.update_block(i, [tx])
             assert_equal(len(b.serialize()), MAX_BLOCK_BASE_SIZE)
-            test1.blocks_and_transactions.append([self.tip, True])
+            blocks.append(b)
             self.save_spendable_output()
             spend = self.get_spendable_output()
 
-        yield test1
+        self.sync_blocks(blocks, True, timeout=180)
         chain1_tip = i
 
         # now create alt chain of same length
         self.move_tip(88)
-        test2 = TestInstance(sync_every_block=False)
-        for i in range(89, LARGE_REORG_SIZE + 89):
-            self.next_block("alt" + str(i))
-            test2.blocks_and_transactions.append([self.tip, False])
-        yield test2
+        blocks2 = []
+        for i in range(1, LARGE_REORG_SIZE + 1):
```

Comment:
Why are you changing this to `1`? We are building blocks on top of block with id `88`, so naming them `"alt89"` ... seems appropriate.